extensions/blocks/podcast-player/templates/podcast-title.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Podcast Title template.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Podcast_Player;
+
+/**
+ * Template variables.
+ *
+ * @var string $title
+ * @var string $link
+ */
+
+if ( ! isset( $title ) ) {
+	return;
+}
+
+if ( isset( $link ) ) :
```

Comment:
Should we check for `! empty` instead, here and in other places where we check if `isset` in the templates?

Reply:
it would be preferable I think